compass/ocean/tests/parabolic_bowl/forward.py

```diff
@@ -0,0 +1,98 @@
+from compass.model import run_model
+from compass.step import Step
+
+
+class Forward(Step):
+    """
+    A step for performing forward MPAS-Ocean runs as part of parabolic bowl
+    test cases.
+    """
+    def __init__(self, test_case, resolution, name,
+                 ramp_type='ramp', coord_type='single_layer',
+                 wetdry='standard'):
+        """
+        Create a new test case
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+
+        resolution : float
+            The resolution of the test case
+
+        name : str
+            the name of the test case
+
+        subdir : str, optional
+            the subdirectory for the step.  The default is ``name``
+
+        coord_type : str, optional
+            vertical coordinate configuration
+
+        ramp_type : str, optional
+            vertical coordinate configuration
+        """
+
+        self.resolution = resolution
+
+        super().__init__(test_case=test_case, name=name)
+
+        self.add_namelist_file('compass.ocean.tests.parabolic_bowl',
+                               'namelist.forward')
+
+        res_name = f'{resolution}km'
+        self.add_namelist_file('compass.ocean.tests.parabolic_bowl',
+                               f'namelist.{res_name}.forward')
```

Comment:
When it's a matter of a single namelist option change, like time step, we've been making those changes with `add_namelist_options`. In this case, we've also been using a config option `dt_per_km` to determine how the time step should change as a function of resolution.